app/models/task_management.rb

```diff
@@ -24,7 +24,7 @@ def self.notifications_count(user_type, id)
 
   def self.all_notifications_for(user_type, id)
     if user_type == "taskee"
-      where(taskee_id: id, status: "inactive").
+      where(taskee_id: id, status: "inactive", paid: true).
```

Comment:
No, a tasker can only create a task without paying. Unless he pays, the taskee will not be notified.